test/parse.test.ts

```diff
@@ -77,6 +77,15 @@ describe('parse', () => {
         expect(!!out.flags.myflag).to.equal(true)
         expect(!!out.flags.force).to.equal(true)
       })
+
+      it('knows "w" is a character', () => {
```

Comment:
I feel we can do without the test, I'll go ahead and remove it

Reply:
sure, of course. i was using that to see it fail and then fixed.